### PR TITLE
[Build/Pkg] Rename spec and manifest files

### PR DESCRIPTION
**Pull request by again4you**
This patch renames spec and manifest files to `mlops-agent.spec` and `mlops-agent.manifest`.

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

#### V4
* Update the include path
* Update the name of unit test package

#### V3
* Rebase the latest main branch

#### V2
* Update the package name with the `mlops-agent`. It can cause the image creation error but it is more clear.
* Keep the name of D-Bus service as `org.tizen.machinelearning.service` since this service also used as Service API.

#### V1
* First Draft
* Update the package daemon name and client library

### CODE REVIEW

File: packaging/mlops-agent.spec [hunk 1]
```diff
@@ -239,20 +239,20 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
 %files
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %license LICENSE
 %attr(0755,root,root) %{_bindir}/machine-learning-agent
 %attr(0644,root,root) %{_unitdir}/machine-learning-agent.service
 %attr(0644,root,root) %config %{_sysconfdir}/dbus-1/system.d/machine-learning-agent.conf
 %attr(0644,root,root) %{_datadir}/dbus-1/system-services/org.tizen.machinelearning.service.service
 
 %files -n libmachine-learning-agent
```
Comment (gichan-jang) [inline packaging/mlops-agent.spec:249]: Don't we need to change package name?

File: packaging/mlops-agent.spec [hunk 2]
```diff
@@ -32,18 +32,18 @@
 %endif
 ###########################################################################
 # Package / sub-package definitions
-Name:		machine-learning-agent
-Summary:	AI service agent for ML(Machine Learning) API and NNStreamer
+Name:		mlops-agent
+Summary:	AI service agent for MLOps and ML(Machine Learning) API
 # Synchronize the version information for ML-Agent.
-# 1. Tizen   : ./packaging/machine-learning-agent.spec
+# 1. Tizen   : ./packaging/mlops-agent.spec
```
Comment (jaeyun-jung) [inline packaging/mlops-agent.spec:38]: need to update line 202 also (machine-learning-agent.spec > mlops-agent.spec)

Comment (again4you) [reply packaging/mlops-agent.spec:38]: ACK!

File: packaging/mlops-agent.spec [hunk 3]
```diff
@@ -263,7 +263,8 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
 %if 0%{?unit_test}
 %if 0%{?release_test}
 %files unittests
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
+%{_bindir}/unittest-ml
```
Comment (jaeyun-jung) [inline packaging/mlops-agent.spec:267]: need removing this line, wook deleted this line on previous PR.

**Review by jaeyun-jung — APPROVED** [review]

File: packaging/mlops-agent.spec [hunk 4]
```diff
@@ -234,27 +234,27 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
+%{_libdir}/libmlops-agent.a
 %{_includedir}/ml-agent/ml-agent-interface.h
-%{_libdir}/pkgconfig/ml-agent.pc
+%{_libdir}/pkgconfig/mlops-agent.pc
 
 %files test
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %license LICENSE
 %attr(0755,root,root) %{test_base_dir}/machine-learning-agent-test
```
Comment (wooksong) [inline packaging/mlops-agent.spec:259]:
```suggestion
%attr(0755,root,root) %{test_base_dir}/mlops-agent-test
```

File: packaging/mlops-agent.spec [hunk 5]
```diff
@@ -234,27 +234,27 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
 %attr(0755,root,root) %{test_base_dir}/services/org.tizen.machinelearning.service.service
```
Comment (wooksong) [inline packaging/mlops-agent.spec:260]: It is required to update this file as well.

File: packaging/mlops-agent.spec [hunk 6]
```diff
@@ -234,27 +234,27 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
 
-%files -n libmachine-learning-agent-devel
-%manifest machine-learning-agent.manifest
-%{_libdir}/libml-agent.so
-%{_libdir}/libml-agent.a
+%files -n libmlops-agent-devel
+%manifest mlops-agent.manifest
+%{_libdir}/libmlops-agent.so
+%{_libdir}/libmlops-agent.a
 %{_includedir}/ml-agent/ml-agent-interface.h
```
Comment (wooksong) [inline packaging/mlops-agent.spec:253]:
```suggestion
%{_includedir}/ml-agent/mlops-agent-interface.h
```

File: packaging/mlops-agent.spec [hunk 7]
```diff
@@ -263,7 +263,7 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
 %if 0%{?unit_test}
 %if 0%{?release_test}
 %files unittests
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %{test_base_dir}/unittests
 %{_libdir}/libml-agent-test.a
 %{_libdir}/libml-agent-test.so*
```
Comment (wooksong) [inline packaging/mlops-agent.spec:269]:
```suggestion
%{test_base_dir}/unittests
%{_libdir}/libmlops-agent-test.a
%{_libdir}/libmlops-agent-test.so*
```

File: packaging/mlops-agent.spec (hunk 6 above)
Comment (again4you) [reply packaging/mlops-agent.spec]: Which one do you perfer? @jaeyun-jung  told me that the second one is better.
- %{_includedir}/ml-agent/mlops-agent-interface.h
- %{_includedir}/mlops/ml-agent-interface.h

File: daemon/include/meson.build [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ml_agent_headers = files('ml-agent-interface.h')
 
 install_headers(ml_agent_headers,
-  subdir: 'ml-agent'
+  subdir: 'mlops-agent'
```
Comment (jaeyun-jung) [inline daemon/include/meson.build:4]: Discussion:
sub-directory to mlops (or other path?)
We now install ml-api headers into nnstreamer, need define path for later work. 
include path : mlops / ml / machine-learning / ...

File: packaging/mlops-agent.spec [hunk 9]
```diff
@@ -234,39 +234,39 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
+%{_libdir}/libmlops-agent.a
+%{_includedir}/mlops-agent/ml-agent-interface.h
+%{_libdir}/pkgconfig/mlops-agent.pc
 
 %files test
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %license LICENSE
-%attr(0755,root,root) %{test_base_dir}/machine-learning-agent-test
+%attr(0755,root,root) %{test_base_dir}/mlops-agent-test
```
Comment (jaeyun-jung) [inline packaging/mlops-agent.spec:259]: need fix 'test_base_dir'

%define test_base_dir %{_bindir}/mlops-agent-test
or
%define test_base_dir %{_bindir}/ml

File: packaging/mlops-agent.spec [hunk 10]
```diff
@@ -234,39 +234,39 @@ install -m 0755 packaging/run-unittest.sh %{buildroot}%{_bindir}/tizen-unittests
...
 
 %if 0%{?unit_test}
 %if 0%{?release_test}
 %files unittests
-%manifest machine-learning-agent.manifest
+%manifest mlops-agent.manifest
 %{test_base_dir}/unittests
-%{_libdir}/libml-agent-test.a
-%{_libdir}/libml-agent-test.so*
+%{_libdir}/libmlops-agent-test.a
```
Comment (jaeyun-jung) [inline packaging/mlops-agent.spec:268]: should we include libmlops-agent-test library here? 

suggestion at line 116 (update require mlops-agent > mlops-agent-test)
%package unittests
....
Requires:	mlops-agent = %{version}-%{release} > mlops-agent-test = %{version}-%{release}

File: daemon/include/meson.build (hunk 8 above)
Comment (jaeyun-jung) [reply daemon/include/meson.build:4]: offline discussion, prefix 'ml'

Comment (again4you) [reply daemon/include/meson.build]: Header files are installed as below.
- /usr/include/ml/mlops-agent-interface.h

File: packaging/mlops-agent.spec (hunk 9 above)
Comment (again4you) [reply packaging/mlops-agent.spec:259]: I am going to use `ml`. Thanks

Comment (wooksong) [reply packaging/mlops-agent.spec:259]: https://github.com/nnstreamer/deviceMLOps.MLAgent/blob/ae046961f187d8adf50d7daa9d562c8c858930be/packaging/mlops-agent.spec#L16

**Review by wooksong — APPROVED** [review]
LGTM!

Let's build with NNS/api:)

**Pull request merged**